modules/andes-core/broker/src/main/java/org/wso2/andes/store/cassandra/HectorBasedMessageStoreImpl.java

```diff
@@ -747,7 +743,7 @@ public int deleteAllMessageMetadataFromDLC(String storageQueueName, String DLCQu
                 throw new AndesException("Error while getting message IDs for queue : " + storageQueueName, e);
             }
         }
-
+        System.out.println("Retrieved " + messageIDs.size() + " number of messages");
```

Comment:
We dont use the system.out.println for logging, can use the logger from sl4j or apache commons to log.